tests/pytorch_tests/model_tests/feature_models/qat_test.py

```diff
@@ -134,3 +138,50 @@ def compare(self, ptq_model, qat_ready_model, qat_finalized_model, input_x=None,
             # check quantization didn't change when switching between PTQ model and QAT ready model
             qat_finalized_output = qat_finalized_model(_in).cpu().detach().numpy()
             self.unit_test.assertTrue(np.isclose(np.linalg.norm(qat_finalized_output - qat_ready_output) / np.linalg.norm(qat_ready_output), 0, atol=1e-6))
+
+
+class QuantizationAwareTrainingMixedPrecisionCfgTest(QuantizationAwareTrainingTest):
+    def __init__(self, unit_test, weight_bits=2, activation_bits=4, finalize=False,
```

Comment:
You don't need to get weight_bits and activation_bits for this test since your not using them (also, not seem to be using weights_quantization_method)